src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotnetLanguage.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+public enum DotnetLanguage
```

Comment:
Nit: should we use `DotNet` instead of `Dotnet`? Applies to multiple places.

Reply:
Renamed